presets/massa-station-preset.js

```diff
@@ -33,20 +33,30 @@ module.exports = {
     createThemes({
       light: {
         primary: colorWhite,
+        primaryAlt: colorDarkBlue,
         secondary: colorLightGrey,
+        secondaryAlt: colorBasicBlue,
```

Comment:
As @mazmassa said this name should change color 
Just for the knowledge

There is two options :
- We define colors and follow DRY concept 
or 
- We define colors for usage and each color has a specific usage 

But now we are doing both 😄

Reply:
We have new colors addressed by Lucas. Let's wait for that first.

Reply:
quick info : secondaryAlt is cHover ou c-hover depending on the final naming convention choice